frontend/javascripts/oxalis/controller/scene_controller.ts

```diff
@@ -271,24 +269,41 @@ class SceneController {
     this.rootNode.remove(skeletonGroup);
   }
 
-  buildTaskingBoundingBox(taskBoundingBox: BoundingBoxType | null | undefined): void {
-    if (taskBoundingBox != null) {
-      if (this.taskBoundingBox != null) {
-        this.taskBoundingBox.getMeshes().forEach((mesh) => this.rootNode.remove(mesh));
+  buildTaskBoundingBox(
+    taskCubeByTracingId: Record<string, BoundingBoxType | null | undefined>,
+  ): void {
+    for (const [tracingId, boundingBox] of Object.entries(taskCubeByTracingId)) {
+      let taskCube = this.taskCubeByTracingId[tracingId];
+      // Remove the old box if it exists
```

Comment:
Although this behavior existed in the "old" code as well, it is very unclear to me why meshes have to be removed and re-added in a function called `buildTaskBoundingBox`. Perhaps we could add more comments explaining this "workaround"(?).

Reply:
good point. I renamed it to updateTaskBoundingBoxes and added a docstring:

```
  /*
     Ensures that a green task bounding box is rendered in the scene for
     each layer.
     The update is implemented by simply removing the old geometry and
     adding a new one. Since this function is executed very rarely,
     this is not a performance problem.
     */
```

Reply:
Great!